File: N Squared Sorts/C++/selectionSort.cpp
void selectionSort(int* arr, int n) {
  for (unsigned int i = 0; i < n - 1; i ++) {
    int min = i;
    for (unsigned int j = i + 1; j < n; j++) {
      if (arr[j] < arr[min]) min = j;
    }
    std::swap(arr[min], arr[i]);
  }
}

File: N Squared Sorts/python/selection_sort.py
def selectionSort(array):
    for i in range(len(array) - 1):
        curMin = i
        for j in range(i + 1, len(array)):
            if array[j] < array[curMin]:
                curMin = j
        if curMin != i:
            array[i], array[curMin] = array[curMin], array[i]
    return array
